src/utils/getPageFromUrl.ts

```diff
@@ -0,0 +1,5 @@
+export default function getPageFromUrl(url: string) {
```

Comment:
here you could also explicitly tell the return type

Reply:
also maybe getPageNumberFromUrl would be little bit more descriptive name for this function. I was little bit confused what this function actually returns before reading the implementation